docs/README.md

```diff
@@ -104,8 +104,8 @@ See [Temporal.ZonedDateTime Documentation](./zoneddatetime.md) for detailed docu
 A `Temporal.PlainDate` object represents a calendar date that is not associated with a particular time or time zone, e.g. August 24th, 2006.
 
 ```js
-const date = Temporal.PlainDate.from({year: 2006, month: 8, day: 24}); // => 2006-08-24
-date.year // => 2006
+const date = Temporal.PlainDate.from({year: 2006, month: 8, day: 24}); // => '2006-08-24'
+date.year // => '2006'
```

Comment:
Yeah, I was hesitant about that, but parsing and generating tests based on the comment RHS got a lot more complicated with mixes of expressions like 2006-08-24 and strings, and I thought it might be worth it as a one time change to allow forward testability - for example, many of the => comments were the result of (previous expression).toString(). Unfortunately, I couldn't just try both and assert that one doesn't throw, because they were often syntax/parser errors.

What do you think? Are the single quotes okay otherwise, or more confusing than they're worth? I can remove them before merging, and keep a local patch to rebase/reapply if I need to re-test. This one is definitely incorrect, though, fixing.

Reply:
It wasn't only this one that was incorrect, but I didn't flag them all when I went through the patch yesterday. For that reason I think I'd really prefer not to have the quotes, because they introduce an opportunity where confusion can slip in.

> many of the => comments were the result of (previous expression).toString()

That's technically true, but I'd intended it more in a future-oriented way, where the text after the `=>` is what a Temporal-aware interactive JS shell would print out. The same way that Date is handled now in Node.js:

```
> new Date()
2021-03-02T22:29:29.219Z
```